unit_testing/myapp/models.py

```diff
@@ -6,5 +7,27 @@ class Person(models.Model):
     first_name = models.TextField(max_length=50)
     last_name = models.TextField(max_length=50)
 
+    cache_key = u'person_{}'
+
     def __unicode__(self):
         return "{name}".format(name=self.first_name+self.last_name)
+
+    def save(self, *args, **kwargs):
+        super(Person, self).save(*args, **kwargs)
+        print("Object is saved")
+
+    def delete(self, *args, **kwargs):
+        cache.delete(self.first_name)
+        super(Person, self).delete(*args, **kwargs)
+
+    @classmethod
+    def get_person(self, fname):
+        cache_key = self.cache_key.format(fname)
```

Comment:
cache_key should have `self.id` as key part so that it should be unique not the `self.first_name` because first name can be repeated.